R/scale-date.r

```diff
@@ -382,6 +382,11 @@ ScaleContinuousDate <- ggproto("ScaleContinuousDate", ScaleContinuous,
   map = function(self, x, limits = self$get_limits()) {
     self$oob(x, limits)
   },
+  get_breaks = function(self, limits = self$get_limits()) {
+    breaks <- ggproto_parent(ScaleContinuous, self)$get_breaks(limits)
+    breaks <- floor(breaks)
```

Comment:
It seems `breaks` can be `NULL` and error here. In that case we can `return(NULL)` early.